test/campaigns_controllers.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
```

Comment:
Is this test meant for all `campaigns` related tests or `campaignInfoById`? We should separate them so one test doesn't do too much, yes?